coincurve/__init__.py

```diff
@@ -1,6 +1,43 @@
-from coincurve.context import GLOBAL_CONTEXT, Context
-from coincurve.keys import PrivateKey, PublicKey, PublicKeyXOnly
-from coincurve.utils import verify_signature
+def load_secp256k1_conda_library():
+    """Load the secp256k1 library from the conda environment."""
+    import warnings
+
```

Comment:
Let's move this to when the exception occurs

Reply:
Yes, I just saw it, working on 2 versions with my nut-sized brain never works well - The gist of it s there so you can continue reviewing, I need to debug why the build does not find the lib - I swear it works fine on my messy branch

Reply:
I don't care at all about failing intermediate commits as long as the final one is green 😄 

I do, as you've seen, care about keeping the import time to an absolute minimum so this library plays nicely with CLIs.

Reply:
Rightly so. Initially, pycharm tooltip recommended to have the imports at the top, so I adopted this as a default, in fact a lot of time I use the tool to find and select the import and it places them at the top
I am glad that I now have a good example of when it is better to fine-tune the imports

It should also be better to just import what is needed, or is it less important, like `from lib import the_fct_I_need`, at least I feel it should have a better memory profile